lib/bot.js

```diff
@@ -390,6 +440,20 @@ class Bot {
       return match;
     });
 
+    // Webhooks first
+    const webhook = this.findWebhook(channel);
+    if (webhook) {
+      logger.debug('Sending message to Discord via webhook', withMentions, channel, '->', `#${discordChannel.name}`);
+      webhook.client.sendMessage(
+        withMentions,
+        {
+          username: author,
+          text,
+          avatarURL: this.getDiscordAvatar(author, channel)
+        }).catch(logger.error);
```

Comment:
Webhook clients aren't attached to the main discord client: 

```this.discord = new discord.Client({ autoReconnect: true });```
vs. 
```this.webhooks[channel] = { [...], client: new discord.WebhookClient(id, token);}```
So there's no way errors will bubble up to the main ```on('error'```

I could probably define a function and use it for both error handling, though.